dbfit-java/core/src/main/java/dbfit/api/vendor/DbStoredProcedureCall.java

```diff
@@ -0,0 +1,16 @@
+package dbfit.api.vendor;
```

Comment:
Maybe, I don't know yet. There also a vendor of the testing framework (that's FitNesse/Fit). It's a bigger topic I think other classes also need to find a better home. So maybe it's better to defer package reorganizations for another PR unless we really benefit from it here.